templates/javascript.js.twig

```diff
@@ -102,3 +102,107 @@ $(document).ready(function() {
     removeSkill();
 });
 {% endmacro %}
+
+{% macro citizenProjectSkills(idSkillsListField, idSkillSearchField ,urlSearch) %}
+$(document).ready(function() {
```

Comment:
I don't think it is needed if the javascript is already at the end of the document no ?

Reply:
No it's needed